include/crocoddyl/core/action-base.hpp

```diff
@@ -87,6 +87,17 @@ class ActionModelAbstractTpl {
   virtual void calc(const boost::shared_ptr<ActionDataAbstract>& data, const Eigen::Ref<const VectorXs>& x,
                     const Eigen::Ref<const VectorXs>& u) = 0;
 
+  /**
+   * @brief Compute the total cost value for nodes that depends only on the state
+   *
+   * It updates the total cost when the next state remains the same to current one. This function is commonly used in
+   * the terminal nodes of an optimal control problem.
```

Comment:
Are there any other use cases of this method besides that of terminal nodes?

Reply:
I don't think so.

Reply:
I will mark this conversation as resolved. Please reopen it if this is the case

Reply:
What I meant with my question is that, if there are no other use cases, instead of writing "this function is commonly used in the terminal nodes" we could write "this function is used in the terminal nodes", because that's the only use case.

Reply:
I see your point. I will improve the documentation as suggested